platform/posix/transport/src/sockets_posix.c

```diff
@@ -143,29 +160,91 @@ static SocketStatus_t resolveHostName( const char * pHostName,
 }
 /*-----------------------------------------------------------*/
 
+static SocketStatus_t connectToAddress( struct sockaddr * pAddrInfo,
+                                        const char * pHostName,
+                                        size_t hostNameLength,
+                                        uint16_t port,
+                                        int tcpSocket )
+{
+    SocketStatus_t returnStatus = SOCKETS_SUCCESS;
```

Comment:
It seems like pHostName is only passed for logging. I suggest that the hostname be logged in the calling code only instead to keep the API signature relevant only to its functionality

Reply:
Agree. Thanks